gnoi_client/gnoi_client.go

```diff
@@ -125,6 +134,25 @@ func killProcess(sc gnoi_system_pb.SystemClient, ctx context.Context) {
 	}
 }
 
+func fileStat(fc gnoi_file_pb.FileClient, ctx context.Context) {
```

Comment:
This is a comment that should be applied to this entire file. 

It seems like we throw panic a lot in this file if gnoi operation fails or during a pre check.

Is that really the intended behavior, as panic will terminate the entire process, unexpectedly shutting down telemetry/gnmi?

If we do decide that panic is the right call, we should wrap them in recover to handle them gracefully as to not cause a crash.

Reply:
Resolving as these panics will not affect gnmi/telemetry server